modyn/supervisor/internal/triggers/performance/decision_policy.py

```diff
@@ -98,22 +102,35 @@ def evaluate_decision(
         mode: TriggerEvaluationMode,
         method: ForecastingMethod,
     ) -> bool:
-        # to compute the rolling average we simply reuse the forecast method, it's not a true forecast here
-        expected = performance_tracker.forecast_optimal_performance(self.config.metric, method="rolling_average")
-        deviation = expected - evaluation_scores[self.config.metric]
-        allowed_absolute_deviation = (
-            self.config.deviation if self.config.absolute else (self.config.deviation * expected)
-        )
+        return self._wrapped.evaluate_decision(measurement=evaluation_scores[self.metric])
```

Comment:
Basically all of those functions are currently just wrappers after the update. In the drift case, we use the decision policies directly. Any reason for keeping those wrappers?

Reply:
the `avoidable misclassification` trigger needs more information; hence I kept the interface. Also I don't have a lot of time for refactoring rn.